src/components/Filter.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { PropTypes } from 'prop-types';
+import { Dropdown } from 'react-bootstrap';
+
+const Filter = ({handleFilter}) => (
+  <Dropdown className='mb-3' onSelect={handleFilter}>
+  <Dropdown.Toggle className="dropdown bg-success">
+    Page Filter
+  </Dropdown.Toggle>
+
+  <Dropdown.Menu className=" bg-info text-center w-auto">
+    <Dropdown.Item eventKey="1" className='border fw-bold fs-3'>1</Dropdown.Item>
+    <Dropdown.Item eventKey="2" className='border fw-bold fs-3'>2</Dropdown.Item>
+    <Dropdown.Item eventKey="3" className='border fw-bold fs-3'>3</Dropdown.Item>
+    <Dropdown.Item eventKey="4" className='border fw-bold fs-3'>4</Dropdown.Item>
+    <Dropdown.Item eventKey="5" className='border fw-bold fs-3'>5</Dropdown.Item>
+    <Dropdown.Item eventKey="6" className='border fw-bold fs-3'>6</Dropdown.Item>
+    <Dropdown.Item eventKey="7" className='border fw-bold fs-3'>7</Dropdown.Item>
+    <Dropdown.Item eventKey="8" className='border fw-bold fs-3'>8</Dropdown.Item>
+  </Dropdown.Menu>
+</Dropdown>
+);
```

Comment:
- [ ] Using the filter breaks the app and returns the error shown in the video [Error video](https://www.loom.com/share/03f80029f77649e992339b4b995e42c3)
 
- [ ] Not quite sure what the functionality of the filter is, kindly explain in your video description